Formula/b/blisp.rb

```diff
@@ -0,0 +1,21 @@
+class Blisp < Formula
+  desc "ISP tool & library for Bouffalo Labs RISC-V Microcontrollers and SoCs "
+  homepage "https://github.com/pine64/blisp"
+  license "MIT"
+  url "https://github.com/pine64/blisp.git",
+      tag:      "v0.0.4",
+      revision: "f601b6b965a922b4a9f400d1c8fd946b69708728"
```

Comment:
Do we need a git clone? Can we use a tarball instead?

Reply:
I tried it with a tarball. But we need submodules to be able to compile it

Reply:
I looked at the way others make builds with submodules